{"name":"As a End User, A Unauthorized user can not login to the application using API","id":"user---operations-about-users---login-\u0026-logout;as-a-end-user--a-unauthorized-user-can-not-login-to-the-application-using-api","testSteps":[{"number":1,"description":"As a End User, A Unauthorized user can not login to the application using API #1: {username\u003dtest-user, password\u003dabc122}","duration":3856,"startTime":"2021-02-12T12:21:23.404+01:00[Europe/Berlin]","exception":{"errorType":"java.lang.AssertionError","message":"Invalid user credentials allowed expected:\u003c401\u003e but was:\u003c200\u003e","stackTrace":[{"declaringClass":"com.serene.tests.features.steps.generic.LoginAPISteps","methodName":"verifyLoginFailure","fileName":"LoginAPISteps.java","lineNumber":55},{"declaringClass":"com.serene.tests.features.steps.stepDefinition.LoginStepDefn","methodName":"login_failed","fileName":"LoginStepDefn.java","lineNumber":64},{"declaringClass":"✽","methodName":"login failed","fileName":"src/test/resources/features/Login/Login.feature","lineNumber":8}]},"result":"FAILURE","precondition":false,"children":[{"number":2,"description":"Given I provide login credentials \"test-user\" and \"abc122\"","duration":611,"startTime":"2021-02-12T12:21:24.022+01:00[Europe/Berlin]","result":"SUCCESS","precondition":false,"children":[{"number":3,"description":"Given user details: /user/login?username\u003dtest-user\u0026password\u003dabc122, test-user, abc122","duration":422,"startTime":"2021-02-12T12:21:24.197+01:00[Europe/Berlin]","result":"SUCCESS","precondition":false}]},{"number":4,"description":"When I send request to login","duration":2532,"startTime":"2021-02-12T12:21:24.635+01:00[Europe/Berlin]","result":"SUCCESS","precondition":false},{"number":5,"description":"Then login failed","duration":86,"startTime":"2021-02-12T12:21:27.168+01:00[Europe/Berlin]","exception":{"errorType":"java.lang.AssertionError","message":"Invalid user credentials allowed expected:\u003c401\u003e but was:\u003c200\u003e","stackTrace":[{"declaringClass":"com.serene.tests.features.steps.generic.LoginAPISteps","methodName":"verifyLoginFailure","fileName":"LoginAPISteps.java","lineNumber":55},{"declaringClass":"com.serene.tests.features.steps.stepDefinition.LoginStepDefn","methodName":"login_failed","fileName":"LoginStepDefn.java","lineNumber":64},{"declaringClass":"✽","methodName":"login failed","fileName":"src/test/resources/features/Login/Login.feature","lineNumber":8}]},"result":"FAILURE","precondition":false,"children":[{"number":6,"description":"Verify login failure: io.restassured.internal.RestAssuredResponseImpl@6535117e, test-user, abc122","duration":73,"startTime":"2021-02-12T12:21:27.169+01:00[Europe/Berlin]","exception":{"errorType":"java.lang.AssertionError","message":"Invalid user credentials allowed expected:\u003c401\u003e but was:\u003c200\u003e","stackTrace":[{"declaringClass":"com.serene.tests.features.steps.generic.LoginAPISteps","methodName":"verifyLoginFailure","fileName":"LoginAPISteps.java","lineNumber":55},{"declaringClass":"com.serene.tests.features.steps.stepDefinition.LoginStepDefn","methodName":"login_failed","fileName":"LoginStepDefn.java","lineNumber":64},{"declaringClass":"cucumber.runner.Runner","methodName":"runPickle","fileName":"Runner.java","lineNumber":80}]},"result":"FAILURE","precondition":false}]}]}],"userStory":{"id":"user---operations-about-users---login-\u0026-logout","storyName":"User - Operations about users - Login \u0026 Logout","path":"src/test/resources/features/Login/Login.feature","narrative":"\tAs a End User, I can login and logout to the application using API","type":"feature"},"featureTag":{"name":"Login/User - Operations about users - Login \u0026 Logout","type":"feature"},"title":"As a End User, A Unauthorized user can not login to the application using API","tags":[{"name":"Login/User   operations about users   login \u0026 logout","type":"feature"},{"name":"Login","type":"capability"}],"startTime":"2021-02-12T12:21:22.864+01:00[Europe/Berlin]","duration":6285,"testFailureCause":{"errorType":"java.lang.AssertionError","message":"Invalid user credentials allowed expected:\u003c401\u003e but was:\u003c200\u003e","stackTrace":[{"declaringClass":"com.serene.tests.features.steps.generic.LoginAPISteps","methodName":"verifyLoginFailure","fileName":"LoginAPISteps.java","lineNumber":55},{"declaringClass":"com.serene.tests.features.steps.stepDefinition.LoginStepDefn","methodName":"login_failed","fileName":"LoginStepDefn.java","lineNumber":64},{"declaringClass":"cucumber.runner.Runner","methodName":"runPickle","fileName":"Runner.java","lineNumber":80}]},"testFailureClassname":"java.lang.AssertionError","testFailureMessage":"Invalid user credentials allowed expected:\u003c401\u003e but was:\u003c200\u003e","testFailureSummary":"FAILURE;java.lang.AssertionError;Invalid user credentials allowed expected:\u003c401\u003e but was:\u003c200\u003e;LoginAPISteps.java","projectKey":"","annotatedResult":"FAILURE","dataTable":{"headers":["username","password"],"rows":[{"values":["test-user","abc122"],"result":"FAILURE"}],"predefinedRows":true,"scenarioOutline":"Given I provide login credentials \"\u003cusername\u003e\" and \"\u003cpassword\u003e\"\n\rWhen I send request to login\n\rThen login failed\n\r","dataSetDescriptors":[{"startRow":0,"rowCount":0,"name":"Invalid"}]},"manual":false,"testSource":"Cucumber","result":"FAILURE"}